app/models/chargeback.rb

```diff
@@ -170,4 +170,8 @@ def self.get_report_time_range(options, interval, tz)
   def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
+
+  def self.chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@simon3z : agree on the placement of the method. @zeari, can you, please, move the method?

@simon3z : that would need a deeper refactoring. Some class(es) that would own the information about the reports that is now hidden in the controller code.